structuredevent-api-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/rest/annotation/TransformGetterType.java

```diff
@@ -0,0 +1,13 @@
+package com.sequenceiq.cloudbreak.structuredevent.rest.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Inherited
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface TransformGetterType {
```

Comment:
there is already a `TransformGetterType` in the codebase, is it possible to reuse that one? same applies for `Immutable`